### DISCUSSION TITLE
Upgrade V2 -> V5

### DISCUSSION THREAD

**Discussion by mertcihankurel** [JedWatson/react-select#5254] [Q&A] [answered]
We are using react-select V2. I need to upgrade it to V5. I'm doing something wrong when upgrading our package version. Do we need to upgrade version by version in order till V5(V2 -> V3, V3 -> V4, V4 -> V5), or we can upgrade directly from V2 to V5?

**Comment by Rall3n** [comment, marked as answer]
Hello @mertcihankurel,

of course you can directly upgrade the dependency to version `5.x.x`, but we recommend you still read through our provided [upgrade guides](https://react-select.com/upgrade) to see important changes throughout every version.

**Comment by mertcihankurel** [comment]
Hey @Rall3n , after upgrading V2-> V5, I'm getting a type error here
<img width="1219" alt="Screen Shot 2022-07-05 at 11 17 37" src="https://user-images.githubusercontent.com/65771498/177299317-c49cac46-26f7-402f-9006-937dd4002fbf.png">
Do you have any idea what may cause this problem?

**Reply by Rall3n** [reply]
@mertcihankurel Make sure you use at least version 3.8 of TypeScript.
